### PR TITLE
Base: Improve BrickGame documentation

### PR DESCRIPTION
**Pull request by kleinesfilmroellchen**
~~I hope the piece names are not copyright infringement.~~

### CODE REVIEW

File: Base/usr/share/man/man6/BrickGame.md [hunk 1]
```diff
@@ -12,4 +12,39 @@ $ BrickGame
...
+
+Z:  x x
+      x x
+
+I:  x x x x
+```
+
+The game will award you points for clearing lines, and the number depends both on the number of lines cleared at once, as well as your current level. The base points are 40, 100, 300, and 1200 for one, two, three, and four lines respectively, each multiplied by your current level number.
+
+The game will transition to a higher level either once you reach a certain score: 1000 points distance during the first ten levels, then 10,000 points distance. For example, you reach level 9 at 9000 points, level 10 at 10,000 points, but level 11 at 20,000 points. Levels determine how fast pieces drop down the screen, making it harder to play on higher levels. While the drop time is over half a second during level 1, it reduces to a tenth of a second at level 14.
```
Comment (GMTA) [inline Base/usr/share/man/man6/BrickGame.md:48]: > The game will transition to a higher level either once you reach a certain score:

That "either" seems misplaced

File: Base/usr/share/man/man6/BrickGame.md [hunk 2]
```diff
@@ -12,4 +12,39 @@ $ BrickGame
 
 ## Description
 
-BrickGame is a classic game.
+BrickGame is a classic game. Pieces consisting of four small squares each fall from the top of the screen, begin fixed in place once they land at the very bottom or on top of other pieces. Once a piece has landed, the next piece will start falling; you can preview this piece on the right side of the screen. By filling an entire row (or line) of squares, that row will be deleted and the rows above will shift down. It is also possible to clear multiple lines at once.
```
Comment (GMTA) [inline Base/usr/share/man/man6/BrickGame.md:15]: > begin fixed in place

"begin" -> "being"

Comment (konradekk) [inline Base/usr/share/man/man6/BrickGame.md:15]: “deleted” → “removed”?

File: Base/usr/share/man/man6/BrickGame.md [hunk 3]
```diff
@@ -12,4 +12,39 @@ $ BrickGame
...
+
+You can control where and how a piece falls, by moving it left or right, making it move down faster, dropping it down instantly, or rotating it left and right. There are multiple control schemes available for these six basic actions; and the space bar always serves as the instant drop key.
+- Left: Move left, Right: Move right, Up: Rotate right, Down: Move down
+- A: Move left, D: Move right, W: Rotate right, E: Rotate left, S: Move down
+- H: Move left, L: Move right, K: Rotate right
+
+The seven pieces are commonly named "T", "J", "L", "O", "S", "Z", and "I". Beware that while "J" and "L" as well as "S" and "Z" are mirrors of each other, they cannot be used interchangeably since you can only rotate pieces.
```
Comment (konradekk) [inline Base/usr/share/man/man6/BrickGame.md:22]: “beware” → “note”?

File: Base/usr/share/man/man6/BrickGame.md [hunk 4]
```diff
@@ -12,4 +12,39 @@ $ BrickGame
...
+S:    x x
+    x x
+
+Z:  x x
+      x x
+
+I:  x x x x
+```
+
+The game will award you points for clearing lines, and the number depends both on the number of lines cleared at once, as well as your current level. The base points are 40, 100, 300, and 1200 for one, two, three, and four lines respectively, each multiplied by your current level number.
```
Comment (konradekk) [inline Base/usr/share/man/man6/BrickGame.md:46]: no need for personification? (i.e. “you will be awarded with points”?)

Comment (kleinesfilmroellchen) [reply Base/usr/share/man/man6/BrickGame.md:46]: I don't think we usually avoid that.

Comment (kleinesfilmroellchen) [reply Base/usr/share/man/man6/BrickGame.md:46]: See e.g. final paragraph of 2048

**Pull request merged**